### PR TITLE
Makefile improvements

### PR DESCRIPTION
**Pull request by sven-urbanski-freiheit-com**
* deleted files Chart.yaml and values.yaml. They are generated! No need to check them in.
* Improved makefile dependencies

### CODE REVIEW

File: charts/kuberpult/Chart.yaml [hunk 1]
```diff
@@ -1,29 +0,0 @@
-apiVersion: v2
```
Comment (mouhsen-ibrahim) [inline charts/kuberpult/Chart.yaml:1]: Why removing this???

Comment (mouhsen-ibrahim) [reply charts/kuberpult/Chart.yaml:1]: In all helm charts that we use `Chart.yaml` and `values.yaml` are defined there

File: charts/kuberpult/Makefile [hunk 2]
```diff
@@ -25,40 +25,42 @@ REL_DIR := $(subst $(TOP_DIR)/,,$(CURDIR))
...
 
 values.yaml: values.yaml.tpl ../../version
 	envsubst < values.yaml.tpl > $@
 
-kuberpult-$(VERSION).tgz: Chart.yaml values.yaml templates/*
+$(TGZ_FILE): Chart.yaml values.yaml templates/*
 	helm package .
 
-ct-test:
+ct-test: Chart.yaml values.yaml
```
Comment (mouhsen-ibrahim) [inline charts/kuberpult/Makefile:46]: You did not add targets to build `Chart.yaml` and `values.yaml` so I think this will fail

**Review by mouhsen-ibrahim — APPROVED** [review]
Not sure if these changes are good on the long run but for now I can approve them

File: charts/kuberpult/Makefile (hunk 2 above)
Comment (sven-urbanski-freiheit-com) [reply charts/kuberpult/Makefile:46]: I didn't add them, because they are already there.

File: charts/kuberpult/Chart.yaml (hunk 1 above)
Comment (sven-urbanski-freiheit-com) [reply charts/kuberpult/Chart.yaml:1]: yes, but they are generated automatically, so we don't need to check them in.

**Pull request merged**